### DISCUSSION TITLE
Retrieving a list of (backup) tasks for a database

### DISCUSSION THREAD

**Discussion by danielwinkler** [ravendb/ravendb#19063] [Q&A] [answered]
I would like to trigger a manual backup of an existing periodic backup task from the client code.

As much as i can see, the studio retrieves the existing tasks from https://instance.ravendb.cloud/databases/database-name/tasks?nodeTag=A

I cannot find a corresponding operation in the client API (similar to `GetServerWideBackupConfigurationsOperation`), which would allow me to identify the backup task id that is required for a `StartBackupOperation`

Am I overlooking something or do we have to write this manually?

**Comment by grisha-kotler** [comment, marked as answer]
Hi @danielwinkler,

You can get the periodic backup task id from the database record.
```
var record = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(store.Database));
var backups = record.PeriodicBackups;
```
